src/python/grpcio_observability/grpc_observability/_gcp_observability.py

```diff
@@ -197,10 +200,12 @@ def save_trace_context(
         trace_options = trace_options_module.TraceOptions(0)
         trace_options.set_enabled(is_sampled)
         span_context = span_context_module.SpanContext(
-            trace_id=trace_id, span_id=span_id, trace_options=trace_options
+            trace_id=trace_id,
+            span_id=span_id,
+            trace_options=trace_options,
+            from_header=True,
```

Comment:
https://opencensus.io/api/python/_modules/opencensus/trace/span_context.html#SpanContextWhat does this do? A quick scan of [the file]() doesn't turn up anything obvious.

Reply:
Can you please provide a reference?

Reply:
I'm taking a similar approach to other propagators (like [BinaryFormatPropagator](https://opencensus.io/api/python/_modules/opencensus/trace/propagation/binary_format.html#BinaryFormatPropagator.from_header)), but since it's an optional field, I'm fine to remove it if you have specific concerns.